boilerplate/openshift/golang-osd-operator/app-sre.md

```diff
@@ -0,0 +1,71 @@
+# Testing APP-SRE Scripts Locally
```

Comment:
These is probably some appsre docs we could like to here.

Reply:
Yes, that would be nice.